app/controllers/concerns/helper.rb

```diff
@@ -0,0 +1,11 @@
+module Helper
+
+  def logged_in?
+    !!session[:id]
+  end
+
+  def current_user
+    User.find(session[:id])
```

Comment:
Ohhhh, damn that makes a lot of sense. I bet that's a _hell_ of a lot more efficient than database lookups. Can't wait to check it out.